### ISSUE TITLE
pTraceShowId doesn't pretty-print

### ISSUE THREAD

**Issue by igrep** [cdepillabout/pretty-simple#19]
`pTraceShowId` (and perhaps the other functions in `Debug.Pretty.Simple`) doesn't seem implemented as I (and maybe you also) expect:

In GHCi,

```hs
> import qualified Data.Map as M
> import Debug.Pretty.Simple
> pTraceShowId $ M.fromList [(1, True)]
fromList "fromList [(1,True)]"
-- ^ Not pretty-printed!
```

although,

```hs
pPrint $ M.fromList [(1, True)]
```

pretty-prints!

```hs
fromList
    [
        ( 1
        , True
        )
    ]
```

This doesn't look consistent behavior!

**Comment by cdepillabout** [comment]
@igrep Thanks a lot for this bug report.  That does look like inconsistent behavior.

I would definitely welcome any PRs fixing this!  And tests to make sure it doesn't happen again would be really great too!

**Comment by cdepillabout** [comment]
Also, pinging @yigitozkavci.  Maybe this was something from #18?

**Comment by yigitozkavci** [comment]
Yes, indeed it looks like it's introduced with #18. Thanks for reporting, I'm investigating this.

**Comment by yigitozkavci** [comment]
I have fixed the issue, and PR is incoming after I implement the tests. Sorry for the inconvenience.

**Comment by cdepillabout** [comment]
This has been fixed in #20, and uploaded to hackage as version 2.0.2.0: https://hackage.haskell.org/package/pretty-simple-2.0.2.0
